test.sh

```diff
@@ -39,9 +39,19 @@ function cleanup()
 
 trap cleanup ERR
 
+# ##################################################################
+# Global variables to support different workloads / configurations to
+# execute client-server performance u-benchmarking tests.
+# ##################################################################
+# Number of messages sent by each client to the server
+NumMsgsPerClient=1000
+
 # Number of clients started for client-server RPC-perf testing.
 NumClients=5
 
+# Server-clock cmdline arg. Default is CLOCK_REALTIME clock
+SvrClockArg=
```

Comment:
Some of the perf test-methods allow you to run `test.sh` overriding these defaults.